update to instructions for local development exercise (#450)
liatrio/devops-bootcamp: docs/3-virtual-machines-containers/3.2-local-development.md

    ?> Many of Vagrant requirements are already taken care of for you in the example. It is up to you to figure out which one(s) you need to add.

2. Add a [Vagrant `post-processor`](https://www.packer.io/docs/post-processors/vagrant/vagrant) to your Packer build template. Configure it to output both a VMWare Fusion image and a Vagrant box.

> - Add a vagrant post processor to automatically upload your box to your cloud acount

Reviewer: ```suggestion
- Add a vagrant post processor to automatically upload your box to your cloud acount
```

Seems odd that this comes before step 3 (create a VagrantCloud account) can we move this towards the end of this exercise and maybe mark it Extra Credit (see [this](https://devops-bootcamp.liatr.io/#/3-virtual-machines-containers/3.3-managing-infrastructure?id=extra-credit) for an example)

Author: I have moved this to extra credit as suggested.

3. Create a VagrantCloud account if needed and upload your box.

4. Read and install required pieces for the [Vagrant VMWare provider](https://developer.hashicorp.com/vagrant/docs/providers/vmware)

> - Hint: use brew to install vagrant-vmware-desktop

Reviewer: ```suggestion
- Hint: For VMWare Utility use `brew install --cask vagrant-vmware-utility`
```

3. Create a VagrantCloud account if needed and upload your box.

4. Read and install required pieces for the [Vagrant VMWare provider](https://developer.hashicorp.com/vagrant/docs/providers/vmware)
- Hint: For VMWare Utility use `brew install --cask vagrant-vmware-utility`

5. Create a Vagrant file that will pull your base box from VagrantCloud

6. Run `vagrant up` and verify Vagrant can pull and start your VM.

7. Verify that `vagrant ssh` will ssh into your VM.

## Extra Credit

>  Add a post processor to automatically upload your vagrant box to Vagrant Cloud.

Reviewer: ```suggestion
 Add a post processor to automatically upload your packer build artifact to Vagrant Cloud.
```
